### PR TITLE
New component structure

### PR DESCRIPTION
**Pull request by marcofucci**
This:
- restructures the components to make them more flexible
- tries to improved the wagtail edit form page with some custom style
- adds an import form which gets the content from the betahealth app and imports it into the db

Things still to address but probably in separate PRs:
- the `rashes-in-babies-and-children` page contains components to redesign (`section_list` and `section_nav`)
- the `hernia/surgery-guide` page contains components to redesign (`article_list` and `article_nav`)
- the `hernia/operation` page contains a `reveal` with a `callout` which causes circle dependency errors
- the API should add a `description` field and the import script should import the related values from the content folder

**Before:**
wagtail - old interface

**After:**
wagtail - new interface

### CODE REVIEW

File: pages/components/text.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+
+class TextBlock(StructBlock):
+    variant = ChoiceBlock(
+        choices=VARIANT_CHOICES
+    )
+    value = RichTextBlock()
+
+    def __init__(self, **kwargs):
+        super(TextBlock, self).__init__(**kwargs)
```
Comment (r4vi) [inline pages/components/text.py]: why not `super().__init__(**kwargs)`

Comment (marcofucci) [inline pages/components/text.py]: 👍  no reason, still getting used to it.

**Comment by r4vi** [comment]
Python is fine, no idea about the other stuff!

File: importer/templates/importer/import_form.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      float: none;
+      width: inherit;
+    }
+  }
+  </style>
+{% endblock %}
+{% block content %}
+    {% include "wagtailadmin/shared/header.html" with title="Import content" %}
+
+    <div class="nice-padding">
```
Comment (tyom) [inline importer/templates/importer/import_form.html]: Is it four or two spaces?

**Comment by teneightfive** [comment]
I think this is fine for this major change of restructuring the components and some of the UI but I imagine we'll have to do some more to the UI to reduce the complexity. We're still requiring content designers to think of components as we do whereas they will be thinking in the context of the page and what content appears in a component. 

We may want to investigate the idea of _presets_ which are related to a certain content structure like "Do/don't" component which gets rendered as two areas of content and the content store handles sending that out via the API as something like a `split_content` with two `split_area`s and possibly a modifier to let the frontend know it needs to render it with the tick and cross lists.

Food for thought...

File: importer/templates/importer/import_form.html (hunk 2 above)
Comment (marcofucci) [inline importer/templates/importer/import_form.html]: @tyom changed to 4 spaces as it seems to be the convention at Torchbox

**Pull request merged**